src/components/organisms/StartProjectPane/RecentProject.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {FC, useState} from 'react';
+
+import Tooltip from 'antd/lib/tooltip';
+
+import {PushpinFilled, PushpinOutlined} from '@ant-design/icons';
+
+import {DateTime} from 'luxon';
+
+import {Project} from '@models/appconfig';
+
+import * as S from './RecentProject.styled';
+
+interface RecentProjectType {
+  project: Project;
+  isActive: boolean;
+  onProjectItemClick?: Function;
+  onPinChange?: Function;
+}
+
+const getRelativeDate = (isoDate: string | undefined) => {
+  if (isoDate) {
+    return DateTime.fromISO(isoDate).toRelative();
+  }
+  return '';
+};
+
+const RecentProject: FC<RecentProjectType> = ({project, isActive, onProjectItemClick, onPinChange}) => {
+  const [isTooltipMessageVisible, setIsTooltipMessageVisible] = useState(false);
+
+  const handleOnProjectItemClick = () => {
+    if (onProjectItemClick) {
```

Comment:
Small trick for the future, though I'd say leave this for now as it's more than good since logic is easy.

The idea is to default optional event handlers to `noop` when you destructure the props. That way you no longer have to worry about it within your component's logic which makes it easier to reason about.

```
import { noop } from "lodash"

const RecentProject: FC<RecentProjectType> = ({
  project,
  isActive,
  onProjectItemClick = noop,
  onPinChange = noop
}) ...
```